### ISSUE TITLE
Separate ringtones for each account

### ISSUE THREAD

**Issue by fulldecent** [64characters/Telephone#97]
Thank you

**Comment by benrubson** [comment]
Yes, would be really nice to be able to choose a specific ringtone (a MP3 file) per account !
Alexei, do you think it could be possible ?
Thank you very much !

**Comment by eofster** [comment]
Yes, I would have it, but it is of a lower priority.

**Comment by fulldecent** [comment]
Please label as <kbd>help wanted</kbd>
